From 5c701da7eb3f9fdd3d9cb410c6e1699f862277ea Mon Sep 17 00:00:00 2001
From: Holger Freyther <zecke@openmoko.org>
Date: Sun, 13 Apr 2008 07:25:57 +0100
Subject: [PATCH] [leds] We would enable PWM for all four timers, disable it for all four as well

Currently we do not use the PWM code to drive the LEDS. We have enabled the PWM
for four timer sources but disable it only for three, fix that up.

Signed-Off-By: Holger Freyther <zecke@openmoko.org>
---
 drivers/leds/leds-neo1973-gta02.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/leds/leds-neo1973-gta02.c b/drivers/leds/leds-neo1973-gta02.c
index 8678670..fbb62c4 100644
--- a/drivers/leds/leds-neo1973-gta02.c
+++ b/drivers/leds/leds-neo1973-gta02.c
@@ -161,6 +161,7 @@ static int __init gta02led_probe(struct platform_device *pdev)
 		case S3C2410_GPB0:
 		case S3C2410_GPB1:
 		case S3C2410_GPB2:
+		case S3C2410_GPB3:
 			lp->has_pwm = 0;
 			s3c2410_gpio_cfgpin(lp->gpio, S3C2410_GPIO_OUTPUT);
 			s3c2410_gpio_setpin(lp->gpio, 0);
-- 
1.5.6.5

